src/views/evcharger/EVChargerControls.vue

```diff
@@ -39,13 +39,13 @@
               </label>
             </div>
           </div>
-          <h4 class="small" v-show="!isSimpleModeRef">Charge Amount: 0xE7</h4>
-          <div class="grid gap-1 align-items-center" style="--bs-columns: 4;" v-show="!isSimpleModeRef">
+          <h4 class="small" v-show="!isSimpleModeRef && isChargingSetShow">Charge Amount: 0xE7</h4>
+          <div class="grid gap-1 align-items-center" style="--bs-columns: 4;" v-show="!isSimpleModeRef && isChargingSetShow">
             <input class="form-control h-100" id="f-ev-charger-charge-amount">
             <label class="g-col-3" for="f-ev-charger-charge-amount">Wh (max {{ evChargerSystemData.evCharger.chargeableElectricity }} Wh)</label>
           </div>
-          <div class="d-grid justify-content-evenly" v-show="!isSimpleModeRef">
-            <button class="btn btn-secondary rounded-pill px-4" type="button" @click="setEVChargerProperties" v-show="!isSimpleModeRef">Set</button>
+          <div class="d-grid justify-content-evenly" v-show="!isSimpleModeRef && isChargingSetShow">
```

Comment:
@justdecodeme ,

Imagine removing the SET button from this part. How would the client set the Operation Mode from then on? 😅